renderdoc/3rdparty/interceptor-lib/README.md

```diff
@@ -46,13 +46,13 @@ cd path/to/llvm
 mkdir build_arm32 # or build_arm64
 cd build_arm32 # or build_arm64
 
-LLVM_ANDROID_ABI=armeabi-v7a # this is arm64-v8a for arm64
-LLVM_TRIPLE=armv8.2a-unknown-linux-android # this is aarch64-unknown-linux-android for arm64
-LLVM_ARCH=ARM # this is AArch64 for arm64
+export LLVM_ANDROID_ABI=armeabi-v7a # this is arm64-v8a for arm64
+export LLVM_TRIPLE=armv8.2a-unknown-linux-android # this is aarch64-unknown-linux-android for arm64
+export LLVM_ARCH=ARM # this is AArch64 for arm64
 
-TBLGEN_PATH=/path/from/above/to/bin/llvm-tblgen
-NDK_PATH=/path/to/ndk16
-TARGET_PATH=/path/to/llvm/install_arm32 # or /path/to/llvm/install_arm64 for arm64
+export TBLGEN_PATH=/path/from/above/to/bin/llvm-tblgen
+export NDK_PATH=/path/to/ndk16
+export TARGET_PATH=/path/to/llvm/install_arm32 # or /path/to/llvm/install_arm64 for arm64
```

Comment:
Not that it matters much either way but I'm curious, why the `export` ? These variables should only be needed locally in the given shell instance.

Reply:
You're right, the cmake command works either way.  I had a typo in one of them earlier and I added the exports during debug.  Uploaded a new PR without them.